_examples/catppuccin.tape

```diff
@@ -1,6 +1,6 @@
 Output catppuccin.gif
 
-Set Theme "Catppuccin Mocha"
+Set Flavour "Catppuccin Mocha"
```

Comment:
this is invalid syntax...

Reply:
Ah yeah right, my bad haven't seen that one ^^